test.js

```diff
@@ -12,7 +12,7 @@ describe('template()', () => {
 	});
 
 	it('should escape `', () => {
```

Comment:
Now the name is incorrect here. A test that would case nesting would be more useful.